new-readme.md

```diff
@@ -81,133 +83,141 @@ npm run dev
 
 <p align="center"><img src="https://github.com/reimagined/resolve/blob/feature/new_readme/readme-top-list-example.png"></p>
 
 * [**todo**](https://github.com/reimagined/resolve/tree/master/examples/todo)
 
-This example demonstrates **simple work with view-models**. You can learn more in [ToDo List App Tutorial](https://github.com/reimagined/resolve/blob/master/docs/Tutorials/ToDo%20List%20App%20Tutorial.md) and [documentation](https://github.com/reimagined/resolve/blob/master/docs/View%20Model.md).
+    This example demonstrates how to work with the [view-models](https://github.com/reimagined/resolve/blob/master/docs/View%20Model.md). The app's creation process is detailed in the [ToDo List App Tutorial](https://github.com/reimagined/resolve/blob/master/docs/Tutorials/ToDo%20List%20App%20Tutorial.md).
 
 <p align="center"><img src="https://github.com/reimagined/resolve/blob/feature/new_readme/readme-todo-example.png"></p>
 
 * [**todo-two-levels**](https://github.com/reimagined/resolve/tree/master/examples/todo-two-levels)
 
-This example demonstrates work with view-models in case of **all events subscribtion**. Sometimes, for example for fast application prototyping, it's necessary to have view-model on client with all server events instead of part. There's a special feature called **wildcard**, that allows to subscribe to all events, you can learn more about that in [documentation](https://github.com/reimagined/resolve/blob/master/docs/View%20Model.md).
+    This example demonstrates how to work with view-models in the **all events subscribtion** case. Learn more about this case in the [View Model](https://github.com/reimagined/resolve/blob/master/docs/View%20Model.md) article. <!-- No info there -->
 
 <p align="center"><img src="https://github.com/reimagined/resolve/blob/feature/new_readme/readme-todo-two-levels-example.png"></p>
 
 ## <a name="documentation">**📚 Documentation**</a>
 
 ### <a name="compatibility">Compatibility</a>
 
-You can comfortably run and develop reSolve application for **Linux**, **Mac** and **Windows** operation systems using **bash** and **cmd** terminals. Also you can develop using **React Native**, but currently it's **experimental** approach.
+You can develop and run **reSolve** applications on **Linux**, **Mac** and **Windows** operating systems. It is also possible to integrate **reSolve** with the **React Native**, but this approach is currently experimental.
 
-You can use any package manager, but we recommend you to look at [**npx**](https://www.npmjs.com/package/npx) and [**yarn**](https://yarnpkg.com/lang/en/) apart from **npm**. We use exactly npx manager in our tutorials like a default way.
+The minimum supported **NodeJS** version is **6.0.0**.
 
-Minimum supported **NodeJS** version is **6.0.0**.
+We recommend using [**npx**](https://www.npmjs.com/package/npx) or [**yarn**](https://yarnpkg.com/lang/en/) instead of **npm**<!--, because ??? -->.
```

Comment:
⚠️

Reply:
because these are faster and easier npm alternatives.

Reply:
I just foresee Dirk's comments. He always requires argumentation in such cases.

Reply:
We can't recommend something without any argumentation. If it's our preference, users don't need to know about it.

Reply:
I wrote
> We recommend using [**npx**](https://www.npmjs.com/package/npx) or [**yarn**](https://yarnpkg.com/lang/en/) instead of **npm** because these are faster and more convenient **npm** alternatives.

Reply:
It's not correct. 

"We don't strongly recommend to use, just to look at those tools. Also a customer can use ANY package manager, that he likes, but we use exactly npx in our tutorials and examples (it more convenient because you can not to install packages before it usage)"

There're many package managers and different tools. We have **three common ways**, and we use the **npx** everywhere by default, we write this approach firstly, because it's our preferences, because the `react` team has the same way. But the customer can ask - why the npx is everywhere? Can I use the npm? Can I use other managers? And we just want to answer to those questions, that's all.

Reply:
> npx is a npm package runner. It's not a npm alternative.
`npx` is a binary in your PATH that you can use INSTEAD of an `npm` binary to do things easier. Though I agree that putting it in the same list with `yarn` does not reflect details perfectly.

>why the npx is everywhere
Not everywhere, we give all commands here for three PMs, and the fact that `npx` is the first in the list speaks about our preferences. I vote for removing this line.